public/v4/apps/langflow.yml

```diff
@@ -0,0 +1,134 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: langflowai/langflow:$$cap_langflow_version
+        volumes:
+            - $$cap_appname-data:/data/langflow
+        restart: always
+        environment:
+            LANGFLOW_SUPERUSER: $$cap_langflow_superuser
+            LANGFLOW_SUPERUSER_PASSWORD: $$cap_langflow_superuser_password
+            LANGFLOW_SECRET_KEY: $$cap_langflow_secret_key
+            LANGFLOW_DATABASE_URL: $$cap_langflow_database_url
+            LANGFLOW_SAVE_DB_IN_CONFIG_DIR: true
+            LANGFLOW_AUTO_LOGIN: $$cap_langflow_auto_login
+            LANGFLOW_BACKEND_ONLY: $$cap_langflow_backend_only
+            LANGFLOW_DEV: $$cap_langflow_dev
+            LANGFLOW_OPEN_BROWSER: $$cap_langflow_open_browser
+            LANGFLOW_HOST: 0.0.0.0
+            LANGFLOW_PORT: $$cap_langflow_port
+            LANGFLOW_AUTO_SAVING: true
+            LANGFLOW_AUTO_SAVING_INTERVAL: 1000
+            LANGFLOW_CACHE_TYPE: async
+            LANGFLOW_LOG_LEVEL: critical
+        caproverExtra:
+            containerHttpPort: $$cap_langflow_port
+caproverOneClickApp:
+    variables:
+        - id: $$cap_langflow_version
+          label: LangFlow Version
+          defaultValue: v1.1.1
+          description: Specify the LangFlow version. Check their Docker page for the latest versions https://hub.docker.com/r/langflowai/langflow/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_langflow_superuser
+          label: Superuser Username
+          defaultValue: admin
+          description: Username for logging into LangFlow. You can edit this field if needed.
+          validRegex: /.{1,}/
+        - id: $$cap_langflow_superuser_password
+          label: Superuser Password
+          defaultValue: $$cap_gen_random_hex(16)
+          description: Password for the superuser. Edit this if needed.
+          validRegex: /.{1,}/
+        - id: $$cap_langflow_secret_key
+          label: Secret Key
+          defaultValue: $$cap_gen_random_hex(32)
+          description: Secret key used to encrypt sensitive data.
+          validRegex: /.{32,}/
+        - id: $$cap_langflow_port
```

Comment:
This doesn't really need to be a variable, let's fix it to 7860. Since it's not mapped and it's internal to the container, it's totally fine to keep it as a fixed value.